README.md

```diff
@@ -88,6 +88,30 @@ sudo /usr/local/share/gcm-core/uninstall.sh
 
 ### Linux Debian package (.deb)
 
+`apt-get` support is available for Ubuntu Bionic Beaver (18.04) and Hirsute 
+Hippo (21.04). Take the following steps to set up and install based on the
+version you are running:
+
+#### Bionic
```

Comment:
I mentioned in your `microsoft/git` PR that these titles might be easier to read in a scan if they were `Ubuntu 18.04 (Bionic)` and `Ubuntu 21.04 (Hirsute)`.